templates/microcloud/contact-us.html

```diff
@@ -0,0 +1,153 @@
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75 is-split-on-medium">
+    <hr class="p-rule">
+    <div class="col">
+      <h2 class="p-text--small-caps">Submit a question</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-6">
+          <form action="https://ubuntu.com/marketo/submit" method="post" id="mktoForm_5364">
+            <p>Tell us about your MicroCloud use case</p>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" maxlength="2000"></textarea>
+              </li>
+              <li class="p-list__item">
+                <label for="firstName" class="is-required">First name</label>
+                <input class="p-form-validation__input" required="" id="firstName" name="firstName" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="lastName" class="is-required">Last name</label>
+                <input class="p-form-validation__input" required="" id="lastName" name="lastName" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="email" class="is-required">Email</label>
+                <input class="p-form-validation__input" required="" id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone" class="is-required">Phone number</label>
+                <input class="p-form-validation__input" required="" id="phone" name="phone" maxlength="255" type="tel" />
+              </li>
+              <li class="p-list__item">
+                <label class="p-checkbox">
+                  <input type="checkbox" aria-labelledby="canonicalUpdatesOptIn" class="p-checkbox__input" value="yes">
+                  <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
+                </label>
+                <p style="padding-left: 2rem;">By submitting this form, I confirm that I have read and agree to Canonical's <a href="https://ubuntu.com/legal/data-privacy">Privacy Policy</a>.</p>
+              </li>
+            </ul>
+            
+            <hr />
+            <button type="submit" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Form', 'eventAction' : 'cloud contact-us', 'eventLabel' : '{{product}}', 'eventValue' : undefined });">Submit form</button>
+
+            <!-- hidden fields -->
+            <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" value="4271" />
```

Comment:
can you add an additional field for `acquisition_url`? This is a new feature we are adding to all non-ubuntu forms so they can get the query strings e.g. https://github.com/canonical/canonical.com/blob/424d172bd4bb198bb2328154b4cd50b4073c7028/templates/data/_data-form.html#L68

Reply:
This should fix the second thing Laszlo mentioned.
For the utm payload you may need to debug the script to check why they are not appended to the request.

<img width="1512" alt="Screenshot 2023-11-02 at 13 24 31" src="https://github.com/canonical/canonical.com/assets/14939793/9e9ebdf3-0554-4e00-8932-833975dbe25c">

Reply:
Thanks Carlos, fixed